js/gallery.js

```diff
@@ -0,0 +1,20 @@
+import {showBigPicture} from './big-picture.js';
+import {renderThumbnails} from './thumbnails.js';
```

Comment:
```suggestion
import {renderThumbnails, similarPublishPhoto} from './thumbnails.js';
```

Импортируй сразу все что нужно из одного файла

Reply:
fixed